csharp_dotnetcore/13.Basic-Bot-Template/BotConfiguration.bot

```diff
@@ -1,7 +1,8 @@
-﻿{
+{
   "name": "BasicBot",

```

Comment:
This needs to snap to version 2.0 bot file schema. 

{
    "name": "echobot-with-counter",
    "services": [
        {
            "type": "endpoint",
            "name": "development",
            "endpoint": "http://localhost:3978/api/messages",
            "appId": "",
            "appPassword": "",
            "id": "1"
        }
    ],
    "padlock": "",
    "version": "2.0"
}